Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -1981,6 +1984,43 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
         let (technologies, tutorials, tutorialArticles, allArticles) = result
         var (otherArticles, rootPageArticles) = splitArticles(allArticles)
         
+        let globalOptions = (allArticles + uncuratedDocumentationExtensions.values.flatMap { $0 }).compactMap { article in
+            return article.value.options[.global]
+        }
+        
+        if globalOptions.count > 1 {
+            let extraGlobalOptionsProblems = globalOptions.map { extraOptionsDirective in
+                let diagnostic = Diagnostic(
+                    source: extraOptionsDirective.originalMarkup.nameLocation?.source,
+                    severity: .warning,
+                    range: extraOptionsDirective.originalMarkup.range,
+                    identifier: "org.swift.docc.DuplicateGlobalOptions",
+                    summary: "Duplicate \(extraOptionsDirective.scope) \(Options.directiveName.singleQuoted) directive",
+                    explanation: """
+                    A DocC catalog can only contain a single \(Options.directiveName.singleQuoted) \
+                    directive with the \(extraOptionsDirective.scope.rawValue.singleQuoted) scope.
```

Comment:
They're all dropped currently but I can update the diagnostic to clarify that. Picking a single one deterministically is somewhat non-trivial and didn't seem worth it to me since it's an unsupported workflow.